src/main/java/com/pi4j/catalog/Application.java

```diff
@@ -1,8 +1,8 @@
 package com.pi4j.catalog;
 
-import com.pi4j.context.Context;
+import java.time.Duration;
```

Comment:
Just a "beauty remark". When using auto save in IntelliJ IDEA with cleanup, imports should get sorted alphabetically. It's not critical at all, but will avoid code changes between commits, where imports only move place.

Reply:
will do a 'optimize imports'